src/query/api/v1/handler/prometheus/native/common.go

```diff
@@ -52,6 +52,7 @@ const (
 	debugParam        = "debug"
 	endExclusiveParam = "end-exclusive"
 	blockTypeParam    = "block-type"
+	lookbackParam     = "lookback"
```

Comment:
Maybe 'lookbackDur' or something similar so it's a little easier to grok what it is?